rust-runtime/aws-smithy-http-server/examples/pokemon_service/src/main.rs

```diff
@@ -37,7 +50,10 @@ pub async fn main() {
     );
 
     // Start the [`hyper::Server`].
-    let server = hyper::Server::bind(&"0.0.0.0:13734".parse().unwrap()).serve(app.into_make_service());
+    let bind: SocketAddr = format!("{}:{}", args.address, args.port)
+        .parse()
+        .expect("Unable to parse the server bind address and port");
```

Comment:
```suggestion
        .expect("unable to parse the server bind address and port");
```

nit: I usually follow the same guidelines for panic messages as for `std::error::Error` https://doc.rust-lang.org/nightly/std/error/trait.Error.html

> Error messages are typically concise lowercase sentences without trailing punctuation